spec/controllers/authentication_controller_spec.rb

```diff
@@ -13,9 +13,17 @@
     it 'creates a new person' do
       expect { subject }.to change { Person.count }.by(1)
     end
-    it 'redirects the person to the root url' do
+
+    it 'redirects the person to the root url if omniauth origin is nil' do
```

Comment:
the `if omniauth origin...` sounds like a context to me. Setting things in the it block maybe not so good.